packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -15,7 +15,7 @@ import {
     HumanReadableAction,
     Network,
     SphinxConfig,
-    DeploymentInfo,
+    FoundryDeploymentInfo,
```

Comment:
Renamed the Solidity `DeploymentInfo` to `FoundryDeploymentInfo` to distinguish it from the TypeScript `DeploymentInfo` object. See the [docs for `FoundryDeploymentInfo`](https://github.com/sphinx-labs/sphinx/pull/1419/files#diff-3b9851f6e1331971546aeb627e15507982bec64d50995404788145fb06cdf709R80-R91) for an explanation